src/languageservice/download.ts

```diff
@@ -199,6 +199,8 @@ export default class ServiceDownloadProvider {
                 .then(inStream => {
                     this.install(inStream, installDirectory).then ( installed => {
                         resolve(installed);
+                    }).catch(installError => {
+                        reject(installError);
```

Comment:
any reason we don't send telemetry event here?  is the error propagated and logged elsewhere?